<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketsListWidget</class>
 <widget class="QWidget" name="SocketsListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6">
   <item>
    <layout class="QHBoxLayout" name="controllLayout">
     <item>
      <widget class="QLabel" name="protocolLabel">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Протокол: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="protocolComboBox">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="tableLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
